chart/values.yaml

```diff
@@ -224,4 +225,15 @@ startupProbe:
     port: http
   failureThreshold: 20
   periodSeconds: 10
-  timeoutSeconds: 5
\ No newline at end of file
+  timeoutSeconds: 5
+
+
+
```

Comment:
Can you remove these extra newlines? There should only be max 1 empty line

Reply:
done on 78bc8a5